.gitignore

```diff
@@ -19,3 +19,7 @@ scripts/glext.h
 *.dir/
 *.sln
 *visualstudio/
+
+# vscode cmake plugin
+build/.cmake
+build/compile_commands.json
```

Comment:
just blacklist the entire folder?

Reply:
fixed, right color was buggy in vscode, i thought some folders were it git :)